bitsdojo_window_linux/linux/bitsdojo_window_plugin.cpp

```diff
@@ -111,8 +111,8 @@ FlBitsdojoWindowPlugin* bitsdojo_window_plugin_new(FlPluginRegistrar* registrar)
 
 void bitsdojo_window_plugin_register_with_registrar(FlPluginRegistrar* registrar) {
   FlBitsdojoWindowPlugin* plugin = bitsdojo_window_plugin_new(registrar);
-  FlView* view = fl_plugin_registrar_get_view(plugin->registrar);
-  enhanceFlutterView(GTK_WIDGET(view));
+  //  FlView* view = fl_plugin_registrar_get_view(plugin->registrar);
+  //  enhanceFlutterView(GTK_WIDGET(view));
```

Comment:
Sorry I had to disable this code for personal reason, it's not ideal to change window parameters unless asked intentionally. I don't know why was this necessary, when we already have a hook in Linux implementation.